### ISSUE TITLE
Minikube v0.23.0 100% CPU usage from kubernetes-dashboard v1.7.0

### ISSUE THREAD

**Issue by bgehman** [kubernetes/minikube#2130]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug Report (and Feature Request to pull in the fix).

Please provide the following details:

**Environment**:

**Minikube version**: v0.23.0
- **OS**: Red Hat Enterprise Linux Workstation release 7.3 (Maipo)
- **VM Driver**: kvm
- **ISO version**: minikube-v0.23.6.iso
- **Install tools**: -na-
- **Others**: -na-

**What happened**:
Minikube VM burns 100% of a CPU

**What you expected to happen**:
Hopefully stay as close to idle as possible.

**How to reproduce it** (as minimally and precisely as possible):
Start a minikube VM with latest v0.23.0 -- run `top` within a `minikube ssh` session and note that dashboard container is burning 100% of a vCPU.

**Output of `minikube logs` (if applicable)**:
-na-

**Anything else do we need to know**:
This is an upstream kubernetes/dashboard bug fixed at: https://github.com/kubernetes/dashboard/issues/2401
Request cutting patch release of Minikube that pulls in kubernetes/dashboard v1.7.1 -- that has the fix.

**Comment by bgehman** [comment]
*Workaround*:
```
$ kubectl get pods -n kube-system -o name | grep dashboard | xargs kubectl delete -n kube-system
```
to bounce the dashboard pod when it gets into the spin-loop 100% CPU state.

**Comment by r2d4** [comment]
You can also disable the dashboard addon with `minikube addons disable dashboard`

We have a PR open to update to 1.7.1, I'll prioritize reviewing that and getting it merged https://github.com/kubernetes/minikube/pull/2112

**Comment by etcusrvar** [comment]
The upstream bug also says to enable heapster. Simply disabling heapster also works.

**Comment by bgehman** [comment]
Ah, that explains why I see this and others don't (I always enable heapster so I can get my resource request/limits accurate).  Thanks!

**Comment by r2d4** [comment]
We've merged 1.7.1 and it will be in the next release.

**Comment by michaelfavia** [comment]
Excellent in the meantime i confirmed that the problem and solution for a workaround work.
